WordPress/build.gradle

```diff
@@ -112,8 +112,6 @@ android {
         testInstrumentationRunner 'org.wordpress.android.WordPressTestRunner'
 
         buildConfigField "boolean", "OFFER_GUTENBERG", "true"
-        buildConfigField "long", "REMOTE_CONFIG_FETCH_INTERVAL", "10"
```

Comment:
Minor (🔍): I guess this should be removed from the `vanilla` product flavor too, right?

Reply:
Good point, thanks! Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/21442/commits/d6b817b2f18eaa5777fb5120091e69ebbdc284ac